apps/docs/content/components/layout/Stack.mdx

```diff
@@ -35,17 +35,17 @@ links:
 ## Examples
 
 ### Reverse
-A stack can reverse the order and direction of its items.
+Stack items order and direction can be reversed.
```

Comment:
```suggestion
The order and direction of stack items can be reversed.
```

Reply:
the other option would be adding an apostrophe to say that it is the order and direction of the items. Otherwise it sound like order is a verb.
```suggestion
Stack items' order and direction can be reversed.
```